appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5252,6 +5252,14 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @Description("Terrain map type")
   String mapTypeTerrain();
 
+  @DefaultMessage("Custom")
+  @Description("Custom map type")
+  String mapTypeCustom();
+
+  @DefaultMessage("CustomUrl")
+  @Description("e.g. https://tile.openstreetmap.org/{z}/{x}/{y}.png")
```

Comment:
Note that this is a description to the translators about how to translate "CustomUrl" into another language, so you may want to explain that here rather than giving an example.

Reply:
Done